desktop/core/src/desktop/js/apps/admin/Overview/Examples.tsx

```diff
@@ -0,0 +1,138 @@
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React, { useState, useEffect } from 'react';
+import LinkButton from 'cuix/dist/components/Button/LinkButton';
+import { DownloadOutlined } from '@ant-design/icons';
+import Loading from 'cuix/dist/components/Loading';
+import {
+  INSTALL_APP_EXAMPLES_API_URL,
+  INSTALL_AVAILABLE_EXAMPLES_API_URL
+} from '../Components/utils';
+import { get, post } from '../../../api/utils';
+import huePubSub from '../../../utils/huePubSub';
+import { i18nReact } from '../../../utils/i18nReact';
+import './Overview.scss';
+
+const exampleAppsWithData = [
+  { id: 'search', name: 'Solr Search', data: ['log_analytics_demo', 'twitter_demo', 'yelp_demo'] }
+];
+
+const excludedApps = ['notebook'];
+
+type InstallExamplesResponse = {
+  status: number;
+  message?: string;
+};
+
+const Examples = (): JSX.Element => {
+  const { t } = i18nReact.useTranslation();
+  const [installingAppId, setInstallingAppId] = useState<string>('');
+  const [availableApps, setAvailableApps] = useState<{ [key: string]: string }>({});
+  const [loading, setLoading] = useState<boolean>(true);
+
+  useEffect(() => {
+    const fetchAvailableApps = async () => {
+      try {
+        const response = await get(INSTALL_AVAILABLE_EXAMPLES_API_URL, {});
+        if (response.apps) {
+          const filteredApps = Object.entries(response.apps)
+            .filter(([appId]) => !excludedApps.includes(appId))
+            .reduce((apps, [appId, appName]) => ({ ...apps, [appId]: appName }), {});
+
+          setAvailableApps(filteredApps);
+        }
+      } catch (error) {
+        console.error('Error fetching available app examples:', error);
+      } finally {
+        setLoading(false);
+      }
+    };
+
+    fetchAvailableApps();
+  }, []);
+
+  const installExamplesCall = async (url: string, data: Record<string, unknown>) => {
+    const response = await post<InstallExamplesResponse>(url, data);
+    return response;
+  };
+
+  const handleInstall = (exampleApp: { id: string; name: string }) => {
+    setInstallingAppId(exampleApp.id);
+    const url = INSTALL_APP_EXAMPLES_API_URL;
+
+    let actualAppName;
+    switch (exampleApp.id) {
+      case 'spark':
+        actualAppName = 'notebook';
+        break;
+      case 'jobsub':
+        actualAppName = 'oozie';
+        break;
+      default:
+        actualAppName = exampleApp.id;
+    }
+
+    try {
+      const appWithExtraData = exampleAppsWithData.find(app => app.id === exampleApp.id);
+      if (appWithExtraData && appWithExtraData.data) {
+        for (const eachData of appWithExtraData.data) {
```

Comment:
Currently you're looping through exampleAppsWithData.data and calling installExamplesCall without await. These are fire-and-forget calls, and any errors inside them won’t be caught. This could lead to unexpected behavior or race conditions.